mungegithub/issue_labeler/simple_app.py

```diff
@@ -14,8 +14,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# pylint:disable=invalid-name, import-error, too-many-locals, deprecated-lambda
```

Comment:
Same deal here. Please remove or else justify. But some of these look like ones we want to fix